content/applications/essentials/activities.rst

```diff
@@ -2,237 +2,348 @@
 Activities
 ==========
 
-*Activities* are follow-up tasks tied to a record in an Odoo database. Activities can be scheduled
-on any page of the database that contains a chatter thread, kanban view, list view, or activities
-view of an application.
+*Activities* are follow-up tasks tied to a record in an Odoo database.
+
+.. _activities/important:
+
+.. important::
+   Throughout this document, the icon that is clicked to create an activity is referred to as an
+   :guilabel:`Activity` icon. This icon changes appearance depending on what, if any, activities are
+   scheduled for that record.
+
+   The :guilabel:`Activity` icon may appear in one of the following ways:
+
+    - :guilabel:`🕘 (clock)`: the default icon, which appears if nothing is scheduled.
+    - :guilabel:`📞 (phone)`: this icon appears if a phone call is scheduled.
+    - :guilabel:`✉️ (envelope)`: this icon appears if an email is scheduled.
+    - :guilabel:`✔️ (checkmark)`: this icon appears if a to-do is scheduled.
+    - :guilabel:`(group of people)`: this icon appears if a meeting is scheduled.
+    - :guilabel:`(arrow pointing up above a line)`: this icon appears if a document is scheduled to
+      be uploaded.
+    - :guilabel:`(pencil in a box)`: this icon appears if a signature request is scheduled.
```

Comment:
I agree- I went with 'importnat' because since this icon could look one of 7 ways, I wanted it clear that it's a changing icon. But I agree with you about an important block being jarring at the begining of a doc =)

I just reformatted what I have since it explained what was happening, and I thought it makes sense.